src/main/resources/application.yml

```diff
@@ -0,0 +1,16 @@
+spring:
```

Comment:
this is where we define all the system properties that are leveraged by spring and our application.  Here initially we have specifications for working with a database.  We set the datasource used by jdbc to communicate with mysql and then we specify jpa configurations to be used by spring data.